src/Moq/Obsolete/MockFactory.cs

```diff
@@ -363,8 +363,18 @@ public virtual void VerifyAll()
 		}
 
 		/// <summary>
-		/// Invokes <paramref name="verifyAction"/> for each mock 
-		/// in <see cref="Mocks"/>, and accumulates the resulting 
+		/// Calls <see cref="Mock.VerifyNoOtherCalls"/> on all mocks created by this factory.
+		/// </summary>
+		/// <seealso cref="Mock.VerifyNoOtherCalls"/>
```

Comment:
Like above, please replace the `cref` value with `Mock{T}.VerifyNoOtherCalls()`.